### DISCUSSION TITLE
Why parameter hierarchy are slicing by default?

### DISCUSSION THREAD

**Discussion by tht-activeviam** [atoti/atoti#748] [Q&A]
Version: 0.7.2
When I create analysis hierarchy, it is slicing hierarchy by default. If I want to see the total, I have to set `hierarchy_name.slicing = False`
What's the reason behind this?
https://docs.atoti.io/latest/lib/atoti/atoti.cube.html#atoti.Cube.create_parameter_hierarchy_from_members

**Comment by rcolle** [comment]
A parameter hierarchy has been designed to allow the user to choose parameters that can then be used in other computations.
Therefore it usually does not make sense to have a grand total across various computational parameters.

What are the use cases for which you want to see the totals? How often do you encounter them?
